process_monitor.py

```diff
@@ -58,8 +50,9 @@ def serve_procmon(port, crash_bin, proc_name, ignore_pid, log_level):
 )
 @click.option("--proc-name", "--proc_name", "-p", help="process name to search for and attach to", metavar="NAME")
 @click.option("--port", "-P", help="TCP port to bind this agent to", type=int, default=DEFAULT_PROCMON_PORT)
-def go(crash_bin, ignore_pid, log_level, proc_name, port):
-    serve_procmon(port=port, crash_bin=crash_bin, proc_name=proc_name, ignore_pid=ignore_pid, log_level=log_level)
+@click.option("--ip", "-I", help="IP of the computer", type=str, default="0.0.0.0")
```

Comment:
We should add the same option to process_monitor_unix.py for consistency.

Can we make the help string of the ip option more obvious? Maybe something like "Listening IP for incoming connections"

Reply:
Another help text suggestion: "Listen on this IP for incoming connections from boofuzz"
Let's do a 127.0.0.1 default for security.

The process_monitor_unix.py changes can be done in this PR, or separately.